documentation/docs/pages/docs/references/rpc-node-providers.mdx

```diff
@@ -33,4 +33,48 @@ You can use [chainlist](https://chainlist.org/) to find all the providers which
 
 ## Local nodes
 
-rindexer should work with any local nodes you run including [Anvil](https://book.getfoundry.sh/anvil/) and [Hardhat](https://hardhat.org/)
\ No newline at end of file
+rindexer should work with any local nodes you run including [Anvil](https://book.getfoundry.sh/anvil/) and [Hardhat](https://hardhat.org/)
+
+# RPC Proxy and Caching
+
+[eRPC](https://erpc.cloud/) is a fault-tolerant EVM RPC proxy and re-org aware permanent caching solution. It is built with read-heavy use-cases in mind such as data indexing and high-load frontend usage.
+
+# Quickstart
+
+1. Create your [`erpc.yaml`](https://docs.erpc.cloud/config/example) configuration file:
+
+```yaml filename="erpc.yaml"
+logLevel: debug
+projects:
+  - id: main
+    upstreams:
+      # You don't need to define architecture (e.g. evm) or chain id (e.g. 137)
+      # as they will be detected automatically by eRPC.
+      - endpoint: https://eth-mainnet.blastapi.io/xxxx
+      - endpoint: https://polygon-mainnet.blastapi.io/xxxx
+      - endpoint: evm+alchemy://xxxx-my-alchemy-api-key-xxxx
+```
+
+See [a complete config example](https://docs.erpc.cloud/config/example) for inspiration.
+
+2. Use the Docker image:
+
+```bash
+docker run -v $(pwd)/erpc.yaml:/root/erpc.yaml -p 4000:4000 -p 4001:4001 ghcr.io/erpc/erpc:latest
+```
+
+3. Send your first request:
```

Comment:
lets remove this send your first request and talk about rindexer aka now you can now set this rpc URl in the network > rpc name https://rindexer.xyz/docs/start-building/yaml-config/networks#rpc and then you are set to go